### PR TITLE
Search Screen New Design & Refactoring

### PR DESCRIPTION
**Pull request by BilalBudhani**
Trello link https://trello.com/c/hTag5y40/87-updated-search-design

### CODE REVIEW

**Comment by BilalBudhani** [comment]
@edencakir @joshmadewell this PR needs review. Sorry for loads of changes, I know it's hard to review in bulk but I had to revamp the whole search screen & eliminate `NativeBase`.

**Comment by edencakir** [comment]
@BilalBudhani Looks good 👍  I'll fetch and review this in detail in an hour

File: src/components/ContentList/styles.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import { StyleSheet } from 'react-native';
+import * as colors from 'kitsu/constants/colors';
+
+export const styles = StyleSheet.create({
+  contentListContainer: {
+    paddingHorizontal: 10,
+    paddingVertical: 10,
```
Comment (edencakir) [inline src/components/ContentList/styles.js]: what about `padding: 10`

File: src/components/Forms/LoginForm.js [hunk 2]
```diff
@@ -59,7 +59,7 @@ const LoginForm = ({ handleChange, data, onSubmit, loading }) => (
       <Button
         block
         disabled={loading}
-        onPress={onSubmit}
+        onPress={() => onSubmit()}
```
Comment (edencakir) [inline src/components/Forms/LoginForm.js]: is there a reason for doing this?

File: package.json [hunk 3]
```diff
@@ -23,7 +24,7 @@
     "prop-types": "^15.5.10",
     "querystring": "^0.2.0",
     "react": "16.0.0-alpha.12",
-    "react-instantsearch": "^4.0.10",
+    "react-instantsearch": "^4.1.0",
```
Comment (edencakir) [inline package.json]: We may need to downgrade this as well, if we move back to 0.46

File: src/components/ContentList/styles.js (hunk 1 above)
Comment (BilalBudhani) [reply src/components/ContentList/styles.js]: `padding` property support has landed in React Native? Sweet! I wasn't aware of that.

File: src/components/Forms/LoginForm.js (hunk 2 above)
Comment (BilalBudhani) [reply src/components/Forms/LoginForm.js]: @edencakir yes. just `onPress={onSubmit}` would send `event` as a first parameter, which is considered as `truthy` value. This makes sure onSubmit is called without any params.

File: package.json (hunk 3 above)
Comment (BilalBudhani) [reply package.json]: I see. I will check and updated it accordingly.

**Comment by BilalBudhani** [comment]
@edencakir good catch. I will fix those white margins and thanks for your appreciation.

**Comment by BilalBudhani** [comment]
@edencakir I fixed the changes you provided inputs on. About those margins, I feel it needs to be kept to embody user's touch experience.

File: src/components/Forms/LoginForm.js (hunk 2 above)
Comment (edencakir) [reply src/components/Forms/LoginForm.js]: @BilalBudhani  Ah I see, never thought this that way

**Comment by BilalBudhani** [comment]
@edencakir can we get this merged?

**Pull request merged**